src/it/resources/com/google/checkstyle/test/chapter5naming/rule53camelcase/InputUnderscoreUsedInNames.java

```diff
@@ -22,95 +22,100 @@ public class InputUnderscoreUsedInNames {
   private final String gradle851 = "";
 
   class InnerBad {
-    String guava_33_4_7 = ""; // violation due to '_' between digit and letter
-    String guava33_4_7 = "";
+    int guava_33_4_7 = 0; // violation, due to '_' between digit and letter, 'must match pattern'
+    int guava33_4_7 = 0;
```

Comment:
can we remove `=0`? to avoid "noise" that is not relevant to validaiton.
please do for all